### PR TITLE
Update building instructions

### PR DESCRIPTION
**Pull request by dio-gh**
Makes the links of the prebuilt LLVM and glslang libs point to the `_mt` version, as the current ones make RPCS3 fail to compile on latest. Thanks to @JohnHolmesII for pointing me to it.

### CODE REVIEW

**Comment by JohnHolmesII** [comment]
"additional libs" has also been moved, to https://github.com/RPCS3/glslang/releases/download/custom-build-win/glslanglibs_mt.7z

**Comment by dio-gh** [comment]
Thanks, included it.

**Comment by JohnHolmesII** [comment]
Well now that you've added 'the', you may as well change the rest.

File: BUILDING.md [hunk 1]
```diff
@@ -110,7 +110,7 @@ git submodule update --init
 
 Open `rpcs3.sln`. The recommended build configuration is `Release - LLVM` for all purposes.
 
-You may want to download precompiled [LLVM libs](https://github.com/RPCS3/llvm-mirror/releases/download/custom-build-win/llvmlibs.7z) and extract to root rpcs3 folder (which contains `rpcs3.sln`), as well as download and extract [additional libs](https://drive.google.com/uc?export=download&id=1A2eOMmCO714i0U7J0qI4aEMKnuWl8l_R) to `lib\%CONFIGURATION%-x64\` to speed up compilation time (unoptimised/debug libs are currently not available precompiled).
+You may want to download the precompiled [LLVM libs](https://github.com/RPCS3/llvm-mirror/releases/download/custom-build-win/llvmlibs_mt.7z) and extract them to the root rpcs3 folder (which contains `rpcs3.sln`), as well as download and extract the [additional libs](https://github.com/RPCS3/glslang/releases/download/custom-build-win/glslanglibs_mt.7z) to `lib\%CONFIGURATION%-x64\` to speed up compilation time (unoptimised/debug libs are currently not available precompiled).
 
 If you're not using precompiled libs, build the projects in *__BUILD_BEFORE* folder: right-click on every project > *Build*.
```
Comment (JohnHolmesII) [inline BUILDING.md]:
```suggestion
If you're not using the precompiled libs, build the projects in *__BUILD_BEFORE* folder: right-click on every project > *Build*.
```

Comment (dio-gh) [reply BUILDING.md]: Would I be smug if I wrote out of scope? :P I can fix it of course, but there's probably countless more mistakes like that, and I can't amend from GitHub directly either, so the commit count will keep increasing like this :D

Might just bite the bullet and clone this branch properly to my hard drive.

Comment (dio-gh) [reply BUILDING.md]: Fixed ;)

**Pull request merged**